core/src/main/java/org/arquillian/smart/testing/configuration/Configuration.java

```diff
@@ -137,6 +152,24 @@ public void setCustomStrategies(String[] customStrategies) {
         return configItems;
     }
 
+    public List<StrategyConfiguration> strategyConfigurations(String[] strategies) {
+        List<StrategyConfiguration> convertedList = new ArrayList<>();
+
+        new JavaSPILoader().all(StrategyConfiguration.class,
+            strategyConfiguration -> Arrays.asList(strategies).contains(strategyConfiguration.name()))
+            .forEach(strategyConfiguration -> {
+                final Class<StrategyConfiguration> aClass1 =
+                    (Class<StrategyConfiguration>) strategyConfiguration.getClass();
+                convertedList.add(mapToObject(aClass1, new HashMap<>(0)));
+            });
+
+        if (convertedList.isEmpty()) {
+            return null;
```

Comment:
Why not return an empty list instead? What extra information `null` gives us?

Reply:
no specific reason.